guides.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<guides links-are-relative="true">
+<guides
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
```suggestion
    xmlns="https://www.phpdoc.org/guides"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

We need to put the namespace or it will not validate